examples/osp-flatcar-aws.yaml

```diff
@@ -381,6 +393,14 @@ spec:
             {{- if or .FlatcarConfig.DisableUpdateEngine .FlatcarConfig.DisableAutoUpdate }}
             systemctl stop update-engine.service
             systemctl mask update-engine.service
+            {{ else if .HTTPProxy }}
```

Comment:
Why the `else` ?  I mean why are you comparing the HTTP Proxy with the auto update?

Reply:
If `update-engine` is disabled then we don't need to update HTTP proxy. Hence, this ` {{ else if .HTTPProxy }}` ensures that we are only add a dropin for `update-engine.service` if it's enabled

Reply:
I see, right, in the old template I guess we were creating this things unconditionally. Cool!